draft-scenarios/coops/close_to_enemies.tex

```diff
@@ -0,0 +1,220 @@
+
+\textbf{Version:} 1.5
+
+\textbf{Source:} \href{https://discord.com/channels/740870068178649108/1313651865648369725}{Archon Studio Discord}
+
+\textit{"Sir! They're here again!"}
+
+\textit{"Sound the alarm, we'll repel them as always!" you command, and go to defend the city. Just as you have done many times before. Such is the hard life of the surrounding clans, with constant wars and skirmishes with nearby neighbors. Fortunately, your alliance is strong, so you know your back is covered.}
+
+\textit{Therefore, the time has come to end the long-standing wars and crush your enemies like a bug once and for all!}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+
+This Scenario is played over 7 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+
+\textbf{Player Count:} 1-6
+
+\textbf{Starting Resources:} 14 \svg{gold}, 5 \svg{building_materials}, 2 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 2 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+\begin{itemize}
+  \item  1 × A Few \svgunit{bronze} Units of your choice
+\end{itemize}
+
+\textbf{Town Buildings:} \svgunit{bronze} Dwelling.
+
+\textbf{Additional Bonus:}: Only base bonus by difficulty (see the main rules).
+
+\textbf{Map Tile Pool:} Take $P$ × random Far (II–III) map Tiles as shared pool. None of them must contain a settlement. ($P$ stands for the number of players)
+
+\subsection*{\MakeUppercase{Map Setup}}
+
+Take the following Map Tiles and arrange them as shown in the Scenario map layout.
+Setup Far (II--III) Map Tiles with settlements of same Faction as players Towns as show a color Cube, or see example.
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tile.
+  \item $P$ × Far (II--III) Map Tile with Settlement of same Faction as players Towns.
+  \item 2$P$ × Far (II--III) Map Tile - must not contain Tiles with Settlements.
+\end{itemize}
+
+\textit{Optional: You can distribute random pieces and covered pieces on the map so that they match your Faction's basic terrain.}
+
+\subsection*{\MakeUppercase{AI Hero Setup}}
+
+Make AI Card Deck for Settlement defenders.
+
+\textbf{Settlement defenders Deck:} 1×Attack Card, 1×Defence Card, 1×Spell Card
+
+\textbf{Settlement defenders Spell Deck:} 1×Magic Arrow
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+Conquer all Settlements by any player.
+
+\subsection*{\MakeUppercase{Defeat Conditions}}
+
+\begin{itemize}
+  \item Any player lose his town.
+  \item All settlements are not owned by any player at the end of 7th turn.
+\end{itemize}
+
+\subsection*{\MakeUppercase{Additional Rules}}
+
+\begin{itemize}
+  \item At any time, Players can spend their Building Token to perform the same actions as if they visited a Trading Post.
+  \item When defending the town, and your Hero is not here, you can still use your Units (for free).
+  \item Each settlement is defend by clans Settlement defenders. (see below)
+\end{itemize}
+
+\subsection*{\MakeUppercase{Timed Events}}
+  \begin{itemize}
```

Comment:
Please remove this nested itemize, there's no need for it. After you do so, we'll work on the text.